lib/vtls/openssl.c

```diff
@@ -1862,6 +1867,8 @@ static CURLcode verifystatus(struct Curl_easy *data,
   ASN1_GENERALIZEDTIME *rev, *thisupd, *nextupd;
   int ret;
 
+  DEBUGASSERT(backend != NULL);
```

Comment:
This is the first of 4 compiler warnings because you add the assert before the last variable declaration:
~~~
vtls/openssl.c:1872:3: error: ISO C90 forbids mixed declarations and code [-Werror=declaration-after-statement]